firestore/templates/homepage.html

```diff
@@ -16,17 +16,18 @@ <h2><a href="{{ url_for('dashboard') }}">Game Dashboard</a></h2>
     <script src="https://www.gstatic.com/firebasejs/4.9.1/firebase-firestore.js"></script>
     <script>
       // Firebase Configuration - REPLACE WITH YOUR OWN VALUES
-      var config = {
-        apiKey: "REPLACE_WITH_YOUR_API_KEY",
-        authDomain: "REPLACE_WITH_YOUR_AUTH_DOMAIN",
-        databaseURL: "REPLACE_WITH_YOUR_DATABASE_URL",
-        projectId: "REPLACE_WITH_YOUR_PROJECT_ID",
-        storageBucket: "REPLACE_WITH_YOUR_STORAGE_BUCKET",
-        messagingSenderId: "REPLACE_WITH_YOUR_MESSAGING_SENDER_ID"
+      var firebaseConfig = {
+        apiKey: "AIzaSyBVyl83gglHHtbUQ5mjUmL5iJkHszjCYL8",
```

Comment:
let's remove these actual values and change back to the "REPLACE_WITH_YOUR_ ... " placeholders

Reply:
Yup. Done.